docker-compose.yml

```diff
@@ -43,3 +43,4 @@ services:
     command:
       - --name=app
       - --playground
+      - --log-level=debug
```

Comment:
we don't need. When app will be deployed to SubQuery the logger level might be switched in runtime.